resources/db/schema/mysql-ddl.sql

```diff
@@ -1087,7 +1087,7 @@ CREATE TABLE user_movies (
   id          INT(16) UNSIGNED               NOT NULL AUTO_INCREMENT,
   user_id INT(16)                        NOT NULL,
   imdbid      MEDIUMINT(7) UNSIGNED ZEROFILL NULL,
-  categoryid  VARCHAR(64)                    NULL DEFAULT NULL,
+  categories_id  VARCHAR(64)                    NULL DEFAULT NULL,
```

Comment:
No. Tested once more, series are added as : 5010|5030|5040 while movies are added as single category (but again that might be a bug). So it definitely should be named categories.